docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2053,303 +2053,285 @@ The following error message codes may appear on the app-server log. Use the foll
+  App extracted but not registered to App Server due to write fail. Error=_FILE_WRITE_MODE_
 
   **Reason:**
 
-  TODO
+  App extracted successfully but not registered to App Server due to write fail. Error=_FILE_WRITE_MODE_.
 
   **Action:**
 
-  TODO
+  Check write access to App Server.
 
 
 
-### ZWED0152W":"Error at call sessionStore. %s: %s
+### ZWED0149W
 
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0153W":"WARNING: CLI Argument missing name or has unsupported type%s
+  Could not find pluginDefinition.json file in App (dir=_AppDir_). Error=_ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Throws _ErrorMsg_ when its not able to find the pluginDefinition.json file in _AppDir_ location.
 
   **Action:**
 
-  TODO
+  No action needed usually, but error may be needed for debugging
+
 
 
+### ZWED0150W
 
-### ZWED0154W":"WARNING: Unrecognized command: %s
+   _identifier_ library path _PluginDefination_ does not exist.
 
   **Reason:**
 
-  TODO
+  It throws warning when library _identifier_ does not exist at path _pluginDefination_.
 
   **Action:**
 
-  TODO
+  Check library exist in the path _pluginDefination_.
+
 
 
+### ZWED0151W
 
-### ZWED0155W":"%s
+  unhandledRejection 
 
   **Reason:**
 
-  TODO
+  When process experiences an unhandledRejection.
 
   **Action:**
 
-  TODO
+  No action needed usually, but error may be needed for debugging
+
 
 
+### ZWED0152W
 
-### ZWED0156W":"1 function initLoggerMessages - ERROR - %s
+  Error at call sessionStore. _APIMethodname_: _Error Object_
 
   **Reason:**
 
-  TODO
+  There is a problem calling a sessionStore _APIMethodname_.
 
   **Action:**
 
-  TODO
+  No action needed usually, but error may be needed for debugging.
+  
+
 
 
+### ZWED0153W
 
-### ZWED0157W":"2 function initLoggerMessages - ERROR - %s
+  WARNING: CLI Argument missing name or has unsupported type _ARG_TYPE_FLAG_/_ARG_TYPE_VALUE_/_ARG_TYPE_JSON_.
 
   **Reason:**
 
-  TODO
+  It throws warning when CLI Argument missing any name or has unsupported type for any one of them _ARG_TYPE_FLAG_/_ARG_TYPE_VALUE_/_ARG_TYPE_JSON_.
 
   **Action:**
 
-  TODO
+  Check any missing argument or unsupported argument.
+
 
 
+### ZWED0154W
 
-### ZWED0158W":"%s
+  WARNING: Unrecognized command: _args_
 
   **Reason:**
 
-  TODO
+  Throws warning error _args_ when command is Unrecognized.
 
   **Action:**
 
-  TODO
+  Check the command once again.
 
 
 
-### ZWED0159W":"Plugin (%s) loading failed. Message: "%s" Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
+### ZWED0155W
+
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for detail to debug.
 
 
 
-### ZWED0160W":"Master cluster not found to save storage for id: %s, value: %s
+### ZWED0156W
+
+  1 function initLoggerMessages - ERROR - _Error_
 
   **Reason:**
 
-  TODO
+  When we encountered an _Error_ ,then 1 function created a logger file with specified message.
```

Comment:
Please read the comments for this one, I commented it
"Attempt to get a log message for a language a user may have specified"
so Reason should be like:
"Attempt to get a log message for a language a user may have specified, has failed with _Error_"